src/events/EventSystem.ts

```diff
@@ -249,7 +249,7 @@ export class EventSystem implements System<EventSystemOptions>
 
         this.setTargetElement(canvas as HTMLCanvasElement);
         this.resolution = resolution;
-        EventSystem._defaultEventMode = options.eventMode ?? 'passive';
+        EventSystem._defaultEventMode = options.eventMode ?? 'auto';
```

Comment:
We don't want to change this back to `auto`, `passive` should be the default behaviour for v8